src/shared/scrapers/AUS/ACT/index.js

```diff
@@ -16,6 +20,20 @@ const getKey = rowLabel => {
   throw new Error(`There is a row we are not expecting: ${lowerLabel}`);
 };
 
+const pivotTheTable = ($trs, $) => {
```

Comment:
They're doing a vertical table for some reason now.

Reply:
uhg